cli/cmd/logs.go

```diff
@@ -97,10 +99,36 @@ func (o *logsOptions) toSternConfig(controlPlaneComponents, availableContainers
 	if err != nil {
 		return nil, err
 	}
+
+	// Based on stern/cmd/cli.go
+	t := "{{color .PodColor .PodName}} {{color .ContainerColor .ContainerName}} {{.Message}}"
+	if o.noColor {
+		t = "{{.PodName}} {{.ContainerName}} {{.Message}}"
+	}
+	funs := map[string]interface{}{
```

Comment:
is this a typo for `funcs`?

Reply:
it is intended to mean `functions`, it comes from https://github.com/oandrew/stern/commit/8723308e46b408e239ce369ced12706d01479532